pluma/pluma-view.c

```diff
@@ -2007,6 +2023,17 @@ pluma_view_button_press_event (GtkWidget *widget, GdkEventButton *event)
 	return GTK_WIDGET_CLASS (pluma_view_parent_class)->button_press_event (widget, event);
 }
 
+static gboolean
+pluma_view_button_release_event (GtkWidget *widget, GdkEventButton *event)
+{
+	if (event->button == 2)
+	{
+		middledown = FALSE;
+	}
+
+	return GTK_WIDGET_CLASS (pluma_view_parent_class)->button_press_event (widget, event);
```

Comment:
Maybe it should be `button_release_event` since we're in similarly named function here?